modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/IngestCommonPlugin.java

```diff
@@ -47,7 +47,7 @@ public IngestCommonPlugin() throws IOException {
         processors.put(SetProcessor.TYPE, new SetProcessor.Factory(parameters.templateService));
         processors.put(AppendProcessor.TYPE, new AppendProcessor.Factory(parameters.templateService));
         processors.put(RenameProcessor.TYPE, new RenameProcessor.Factory());
-        processors.put(RemoveProcessor.TYPE, new RemoveProcessor.Factory(parameters.templateService));
+        processors.put(RemoveProcessor.TYPE, new RemoveProcessor.Factory());
```

Comment:
it would be best to leave templatable field values, since that was the previously supported behavior